data/course.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "Your course title",
+  "description": "Your course description",
+  "version": "0.1.0",
+  "authors": ["@dislearning"],
+  "start": "presentation",
+  "toc": [
+    {
+      "type": "text",
+      "slug": "presentation",
+      "title": "Presentation"
+    },
+    {
+      "type": "section",
```

Comment:
I like to add sections

Reply:
This is only a test, we need to define a good way to represent the structure of the course, so this is only for have a different element type.